package.json

```diff
@@ -11,6 +11,7 @@
   },
   "scripts": {
     "build": "tsc",
+    "dev": "tsc -w",
```

Comment:
We can alias this to something other than `dev` if it makes more sense, just thought it would be nice to have an alias. (Also, I know that you can just run `yarn build -- -w`, but if we're aliasing `tsc`, may as well alias `tsc -w`).

Reply:
Seems fine.  I think "watch" would also work.  Whatever we settle on can you add a reference to this in the README with the other build commands?

Reply:
`watch` seems more descriptive to me.